tasks-dev/assemble.js

```diff
@@ -65,11 +65,11 @@ gulp.task('assemble', () => {
         // stops watch from breaking on error
         .pipe(plumber(config.gulp.onError))
 
-        // can’t get newer to work with page includes: such that a many > 1 relationship.  Commenting out so just recompiles all for now
-        // .pipe(newer({
-        //     dest: pathBuilder.docsDistDir,
-        //     ext: '.html'
-        // }))
+    // can’t get newer to work with page includes: such that a many > 1 relationship.  Commenting out so just recompiles all for now
+    // .pipe(newer({
+    //     dest: pathBuilder.docsDistDir,
+    //     ext: '.html'
+    // }))
 
```

Comment:
IMHO, we should either remove the comments and raise a Jira to fix it, or investigate why it isn't working and fix it as part of this ticket (this comment only exists because you let a white-space change creep into the PR!)